spec/services/persist_derivatives_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe CurationConcerns::PersistDerivatives do
+  before do
+    allow(CurationConcerns.configuration).to receive(:derivatives_path).and_return('tmp')
+  end
+
+  describe '.output_file' do
+    subject { described_class.output_file(object, destination_name, &block) }
+
+    let(:object) { double(id: '123') }
+    let(:destination_name) { 'thumbnail' }
+
+    # rubocop:disable Style/Lambda
+    let(:block) { lambda { true } }
```

Comment:
OK, then I'd be in favor of following the pattern I just described above: explicitly excluding the spec directory from the Style/Lambda cop in .rubocop.yml vs. using disable/enable comments. Is that :cool: ?